rpmbuild/main.py

```diff
@@ -301,10 +301,8 @@ def dump_configs(args, config):
     resultdir = config.get("main", "resultdir")
     builder = MockBuilder(task, None, resultdir, config)
 
-    configdir = os.path.join(resultdir, "configs")
-    config_paths = builder.prepare_configs(configdir)
-    for config_path in config_paths:
-        log.info("Wrote: "+config_path)
+    builder.prepare_configs()
+    log.info("Wrote: "+builder.mock_config_file)
```

Comment:
I know we had the mistake already in our code but since you are touching it, could you please change 

```
log.info("Wrote: "+builder.mock_config_file)
```

to 

```
log.info("Wrote: %s", builder.mock_config_file)
```

That's how logging parameters should be passed.

Reply:
Sure thing!